packages/cli/src/commands/ui/create/vue.ts

```diff
@@ -72,8 +72,8 @@ export default class Vue extends Command {
     const {providerUsername} = await this.getUserInfo();
 
     const cliArgs = [
-      'invoke',
-      '@coveo/vue-cli-plugin-typescript',
+      'add',
+      '@coveo/typescript',
```

Comment:
Since the package now respects the Vue-cli plugin standard, we can use the short version, which is pretty cool.